orbit/pkg/packaging/wix/wix.go

```diff
@@ -108,22 +101,26 @@ func Light(path string, native bool) error {
 		args = append(
 			args,
 			"docker", "run", "--rm", "--platform", "linux/amd64",
-			"--volume", path+":"+path, // mount volume
+			"--volume", path+":/wix", // mount volume
 			"fleetdm/wix:latest", // image name
 		)
 	}
 
 	args = append(args,
-		"light", windowsJoin(path, "heat.wixobj"), windowsJoin(path, "main.wixobj"), // command
+		"light", "heat.wixobj", "main.wixobj", // command
```

Comment:
Ah, this works because `/wix` is set to the work dir right when `!native`, right?

Reply:
yes, exactly!